scripts/build.py

```diff
@@ -1681,6 +1681,10 @@ def release(self):
             body = self._get_github_release_body()
             release_name = 'v%s - %s' % (self.version, engine_channel or self.channel)
             release_to_github.release(self, tag_name, release_sha1, releases[0], release_name=release_name, body=body, prerelease=prerelease, editor_only=is_editor_branch)
+        
+        # Release to steam for stable only
+        # if tag_name and (self.channel == 'editor-alpha'):
+        #     self.release_to_steam()
```

Comment:
I thought about once a month release (push to master) but now I think maybe you are right and do it the same way as we do for internal updates - makes sense

Reply:
My thinking is that we automatically release to Steam when we release a normal stable release (this is a relatively recent change that we automatically release to stable). Although I've decided to disable it for now since I haven't had time to test the release flow manually on a Linux machine yet. So the idea for now is that we do the release to steam as a manual step and the end of the release flow.